packages/app/README.md

```diff
@@ -197,6 +197,29 @@ parameters to send to the server. If you use one of H5Grove's default API
 implementations, then you'll need to use this prop to pass the `file` query
 parameter as shown above.
 
+#### `getExportURL?: URL | (() => Promise<URL | Blob>) | undefined`
+
+The `DataProvider#getExportURL` method is used by the toolbars to generate URLs
+for exporting the current dataset/slice in various formats. This prop allows
+providing your own implementation of this method. If you don't or provide a
```

Comment:
I would put the sentence beginning by `If you don't ...` at the end of the paragraph.

The doc of the other providers point to this paragraph so it feels odd to have a specificity of `H5GroveProvider` described in the middle of it.

Reply:
Also I found the phrasing `If you don't or provide a function that handles only specific export scenarios` confusing.

Suggestion:
```
`H5GroveProvider` falls back to generating URLs based on the `/data` endpoint and `format` query param if you don't provide your own implementation 
```

Reply:
I've reworded the paragraph you mention, as well as the text in the other providers.